frontend/src/components/modals/types.ts

```diff
@@ -90,6 +91,11 @@ type OwnProps = {
   [ModalTypes.JOIN_PROJECT_MODAL]: { invitation: Invitation };
   [ModalTypes.JOIN_LINK_INVALID_MODAL]: {};
   [ModalTypes.JOIN_LINK_NO_ACCESS_MODAL]: { invitationLink: string };
+  [ModalTypes.CHANGE_PASSWORD_MODAL]: { id: number };
+  [ModalTypes.INFO_MODAL]: { header: string; content: InfoModalContent };
+  [ModalTypes.LEAVE_ROADMAP_MODAL]: {
+    roadmapId: number;
+  };
   [ModalTypes.CONFIRM_PASSWORD_MODAL]:
```

Comment:
Keeping the unnecessary changes to a minimum helps make sense of the actual change.

In this case, moving the "confirm password modal" stuff back above the new "change password modal" lines would help. (same in `ModalRoot`)